logentries/templates/daemon-set.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
```

Comment:
maybe just name the file `daemonset.yaml` (without the dash)?

Reply:
https://github.com/commercetools/k8s-charts/pull/2/commits/c25f1027d8bb139f493c19709d1e86454b243463